Update script.js

HannaDaunar opened the pull request:
added js-code that display predictions

HannaDaunar committed:
Update script.js

added js-code that display predictions

mdubik777 commented on script.js:
```diff
@@ -13,3 +13,45 @@
+}
+
+function makeForecastByTemplate(text, probability) {
+    const forecastItemTemplate = document.querySelector('#forecast-item');
```
Аня,  выборки лучше писать сверху, 1 раз это запишется в переменную и будет использоваться. А так каждый раз при вызове функции будет снова и снова делаться одна и та же выборка

mdubik777 commented on script.js:
```diff
@@ -13,3 +13,45 @@
+const sendButton = document.querySelector('.forecast-btn');
+sendButton.addEventListener('click', function() {
+
+    let predictionNumber = generateRandomValue(1, 5);
```
А почему let? predictionNumber явно в коде не перезаписывается

HannaDaunar committed:
Update script.js

changed the place of const forecastItemTemplate and changed let to const for predictionNumber
